yew/src/virtual_dom/vtag.rs

```diff
@@ -821,11 +820,11 @@ mod tests {
 
     #[test]
     fn filter_empty_string_classes() {
-        let a = html! { <div class=vec![""]></div> };
-        let b = html! { <div class=("", "")></div> };
-        let c = html! { <div class=""></div> };
+        let a = html! { <div class?=(vec![""])></div> };
+        let b = html! { <div class?=("", "")></div> };
+        let c = html! { <div class?=("")></div> };
         let d_arr = [""];
-        let d = html! { <div class=&d_arr[..]></div> };
+        let d = html! { <div class?=(&d_arr[..])></div> };
```

Comment:
This now works \o/ But you need to specific that class is optional and you must use the parenthesis